eventActions.js

```diff
@@ -535,10 +597,28 @@ ActionSetProp.prototype.fromJS = function (data) {
  * @returns {object}
  */
 ActionSetProp.prototype.toJS = function () {
+    var origChanges = this.refToObjectProperty();
+    var refToObjectProperty = [];
+    for (var i = 0; i < origChanges.length; i++) {
+        var obj = origChanges[i].toJS();
+        refToObjectProperty.push(obj);
+    }
+    var targetId = null;
+    if (this.target) {
```

Comment:
why target? is not defined...

Reply:
Done , I removed that there is no need for that.